### PR TITLE
[stable13] [LDAP] The WebUI Wizard also should not assign empty config IDs

### PR DESCRIPTION
**Pull request by blizzz**
backport of #10820 

@MorrisJobke not so serious, feel free to move to 13.0.7. This should not be a blocker.

### CODE REVIEW

**Comment by MorrisJobke** [comment]
RC 1 is out -> moving to 13.0.7

**Comment by blizzz** [comment]
failing tests not related

**Comment by blizzz** [comment]
reviews, pozhaluysta? :smile_cat:

File: apps/user_ldap/lib/Settings/Admin.php [hunk 1]
```diff
@@ -49,6 +49,14 @@ public function __construct(IL10N $l) {
 	public function getForm() {
 		$helper = new Helper(\OC::$server->getConfig());
 		$prefixes = $helper->getServerConfigurationPrefixes();
+		if(count($prefixes) === 0) {
```
Comment (nickvergessen) [inline apps/user_ldap/lib/Settings/Admin.php]: Use empty in the future ;)

**Review by nickvergessen — APPROVED** [review]
:elephant:

File: apps/user_ldap/lib/Settings/Admin.php (hunk 1 above)
Comment (blizzz) [reply apps/user_ldap/lib/Settings/Admin.php]: it wouldn't be a big array in this case, still you're right.

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**